nginx.sh

```diff
@@ -7,4 +7,21 @@ else
   sudo apt install nginx
 fi
 
+# This worked for me on my system, but we should test...
+# Then wrap all code, in idempotent code blocks
 
+# Install build dependencies
+
+sudo apt install build-essential libpcre3 libpcre3-dev libssl-dev
```

Comment:
We don't need to "build" nginx from source, just install it from package using apt.

Reply:
By prompted, I mean that was the error I was getting. But I could have botched running nginx.sh somehow.